2008-11-06  Michael Tautschnig  <mt@debian.org>

	* lib/setup-storage/Exec.pm: Use --timeout=10 instead of --timeout 10 to stay
		compatible with etch (thanks Vincent Fortier)
Index: trunk/lib/setup-storage/Exec.pm
===================================================================
--- trunk.orig/lib/setup-storage/Exec.pm
+++ trunk/lib/setup-storage/Exec.pm	
@@ -246,7 +246,7 @@
 ################################################################################
 sub execute_with_udevsettle {
   my ($command, $stdout, $stderr) = @_;
-  return &execute_command("udevsettle --timeout 10 && $command", $stdout,
+  return &execute_command("udevsettle --timeout=10 && $command", $stdout,
     $stderr);
 }
 
